rclcpp_lifecycle/src/lifecycle_node.cpp

```diff
@@ -114,6 +115,13 @@ LifecycleNode::LifecycleNode(
       options.use_clock_thread()
     )),
   node_waitables_(new rclcpp::node_interfaces::NodeWaitables(node_base_.get())),
+  node_builtin_executor_(new rclcpp::NodeBuiltinExecutor(
+      node_base_,
+      node_topics_,
+      node_services_,
+      node_logging_,
```

Comment:
?

Reply:
Yes. I have removed it before pushing to github rclcpp